### PR TITLE
Decorating compute resource tests according to defect

### PR DESCRIPTION
**Pull request by oshtaier**

### CODE REVIEW

File: tests/foreman/ui/test_docker.py [hunk 1]
```diff
@@ -966,6 +967,7 @@ def setUpClass(cls):
         super(DockerComputeResourceTestCase, cls).setUpClass()
         cls.organization = entities.Organization().create()
 
+    @skip_if_bug_open('bugzilla', 1336727)
```
Comment (abalakh) [inline tests/foreman/ui/test_docker.py]: Could you please sort decorators in alphabetical order? Not a blocker

**Comment by abalakh** [comment]
ACK

**Comment by oshtaier** [comment]
Please, go ahead and merge

**Comment by sthirugn** [comment]
ACK

**Pull request merged**